cqlsh/cqlsh.py

```diff
@@ -116,20 +112,14 @@ def find_zip(libprefix):
     ver = os.path.splitext(os.path.basename(cql_zip))[0][len(CQL_LIB_PREFIX):]
     sys.path.insert(0, os.path.join(cql_zip, 'cassandra-driver-' + ver))
 
-third_parties = ('futures-', 'six-', 'geomet-')
+# the driver needs dependencies
+third_parties = ('six-', 'pure_sasl-')
```

Comment:
Unrelated to this PR: The driver hasn't dropped `six` yet? Weird...

Reply:
Actually, grep'ing the `cqlshlib` code, it looks like `six` was removed everywhere... 

I assume this code still matches upstream, in which case we should keep it, but maybe upstream should be updated to drop `six-` from `third_parties`? I dunno, I haven't actually looked at this much...